pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -161,42 +161,42 @@
          </item>
          <item>
           <property name="text">
-           <string>Weave (Top Field First, Sawtooth)</string>
+           <string extracomment=":I18N COMMENT: Weave: deinterlacing method that can be translated or left as-is in English.">Weave (Top Field First, Sawtooth)</string>
           </property>
          </item>
          <item>
           <property name="text">
-           <string>Weave (Bottom Field First, Sawtooth)</string>
+           <string extracomment=":I18N COMMENT: Weave: deinterlacing method that can be translated or left as-is in English.">Weave (Bottom Field First, Sawtooth)</string>
           </property>
          </item>
          <item>
           <property name="text">
-           <string>Bob (Top Field First, Full Frames)</string>
+           <string extracomment=":I18N COMMENT: Bob: deinterlacing method that refers to the creator's name.">Bob (Top Field First, Full Frames)</string>
```

Comment:
I thought "Bob" actually referred to Bob Weaver, a creator of the deinterlacer effect, here.

Reply:
![image](https://user-images.githubusercontent.com/24227051/221939019-c9317589-53b9-4488-8e69-99dfc194567b.png)

Bob is pretty close to getting progressive visuals without being progressive instead of blending which makes it blurry but also hides issues.

Reply:
I just checked that Mr. Weaver is actually the creator of another deinterlacing effect called BWDIF. It's not bobbing, then.

Reply:
(Reference: http://avisynth.nl/index.php/BWDIF )